scripts/setup_gradient_workflow.py

```
@@ -0,0 +1,234 @@
+        """Run the workflow."""
+        if not self.validate_workflow_file():
+            return False
+
+        cmd = ["gradient", "workflows", "run", str(self.workflow_file)]
+
+        if project_id:
+            cmd.extend(["--projectId", project_id])
+
+        try:
+            print(f"🚀 Running workflow: {' '.join(cmd)}")
+            result = subprocess.run(cmd, check=True)
```

Ruff (0.12.2)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)